packages/holocron/src/holocronModule.jsx

```diff
@@ -96,33 +96,36 @@ export default function holocronModule({
   mergeProps,
   options = {},
 } = {}) {
+  let that;
   return function wrapWithHolocron(WrappedComponent) {
     class HolocronModuleWrapper extends React.Component {
       constructor(props) {
         super(props);
         this.state = {
           loadCount: 0,
           status: 'loading',
+          currentProps: props,
         };
         this.mounted = false;
+        that = this;
```

Comment:
Cant this be avoided by using an arrow function ?

Reply:
It is needed for `executeLoadingFunctions`, which expects instance of class.